### DISCUSSION TITLE
Free Clown Healing

### DISCUSSION THREAD

**Discussion by LifeLikeSage** [FakeFishGames/Barotrauma#14904] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I'm not sure if this is a bug (you can be the ultimate deciders on the matter) but clowns have found an abuse where they suffocate each other with suits with no oxygen and clown hammer one another to get infinite healing.
Maybe have the healing from the clown hammer healing talent only heal those who are actually dying from injuries and not those who are suffocating?

### Reproduction steps

((This is technically a mechanical abuse of the game))

1. Get a clown with the talent that allows healing with the hammer of those who are dying/unconscious.
2. Put a suit with no oxygen on someone who is injured.
3. Wait for them to pass out.
4. Wail on them with the clown hammer to heal them.
5. repeat from 2 to 4 until fully healed with no resources paid.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

We feel this is an entirely valid use of the talent: the description of the talent says it allows reviving unconscious characters with the toy hammer, there's no condition that the unconsciousness can't have been induced intentionally. I agree this is a pretty powerful effect, but in my opinion it should be, considering you unlock it with the final talent in the clown talent tree.
